templates/client-daemonset.yaml

```
@@ -191,7 +191,7 @@ spec:
                 -node-meta=pod-name:${HOSTNAME} \
                 -node-meta=host-ip:${HOST_IP} \
                 -hcl='leave_on_terminate = true' \
-                -disable-host-node-id=false \
+                -disable-host-node-id={{ .Values.client.disableHostNodeID | default false }} \
```

```suggestion
                -disable-host-node-id={{ .Values.client.disableHostNodeID }} \
```
Thanks for adding the tests! It's looking great! We can shorten this since the helm value defaults to false. But after that this should be good to go.

---

Done